p2panda-rs/src/entry/error.rs

```diff
@@ -31,6 +31,10 @@ pub enum EntrySignedError {
     #[error("operation needs to match payload hash of encoded entry")]
     OperationHashMismatch,
 
+    /// Handle errors when encoding operation.
+    #[error("error occured when encoding operation")]
+    OperationEncodingError(String),
```

Comment:
Suggest renaming this to something that's less similar to `OperationEncodedError`.